llvm/test/tools/sycl-post-link/spec-constants/default-value/device-image.ll

```diff
@@ -0,0 +1,90 @@
+; Test checks the content of simple generated device image.
+; It checks scalar, sret and "return by value" versions of SpecConstant functions.
+; Also test checks generated symbols.
+
+; RUN: sycl-post-link -split=auto -spec-const=rt -symbols -S -o %t.table %s -generate-device-image-default-spec-consts
+; RUN: FileCheck %s -input-file %t.table -check-prefix=CHECK-TABLE -DPATH=%t
+; RUN: FileCheck %s -input-file %t_0.prop -check-prefix=CHECK-PROP0
+; RUN: FileCheck %s -input-file %t_1.prop -check-prefix=CHECK-PROP1
+; RUN: FileCheck %s -input-file %t_0.ll -check-prefix=CHECK-IR0
+; RUN: FileCheck %s -input-file %t_1.ll -check-prefix=CHECK-IR1 --implicit-check-not "SpecConstant"
+; RUN: FileCheck %s -input-file %t_0.sym -check-prefix=CHECK-SYM0
+; RUN: FileCheck %s -input-file %t_1.sym -check-prefix=CHECK-SYM1
+
+; CHECK-TABLE: [[PATH]]_0.ll|[[PATH]]_0.prop|[[PATH]]_0.sym
```

Comment:
Note: just using `{{.*}}_0.ll` should be enough here. I guess that checking the exact path is not the main purpose of the check, but checking how many lines there are in the table is.